lms/djangoapps/ucsd_registration_form/admin.py

```diff
@@ -0,0 +1,29 @@
+import csv
```

Comment:
Seems good, I just want to add a little suggestion that it will be better to move this app under **ucsd_features** (openedx/features/ucsd_features) folder as we try to write all our custom code or features in one place to track all changes easily.

Reply:
thanks for the suggestion - matthew is looking into moving this